app/assets/config/manifest.js

```diff
@@ -1,3 +1,2 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
```

Comment:
When we use esbuild, it automatically sends the stylesheets to the builds folder. However, to ensure that flatpickr's styles get picked up, we import them in the application.js

This can cause a potential sprockets doublelink error. Therefore, I removed the stylesheets from the manifest.js, which means we will need to explicitly require the css of additional js dependencies if
they have them.